content/rule_book/roles_and_responsibilities.md

```diff
@@ -42,14 +42,14 @@ The Quizmaster must:
     - On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
     - On a Q/Q2V, CVR/MACVR, CR/MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
         - Note: On a CVR/MACVR, CR/MACR Question once the quizzer has provided all the information in both the question text and answer the quizmaster will say "What is your question", if the response to the reference question supplied by the quizzer is "More", the quizzer should consider whether the quizmaster had completely pronounced the chapter or verse number.
-    - On a Reference type question, if the quizzer gives a reference question that is different than the one on the card, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the one on the card.
+    - On a Reference type question, if the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
     - Consult with the answer judge(s) at any time he or she feels is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
     - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
 
 ### Answer Judges (AJ)
 
 - All conferring among quizmaster and answer judge(s) shall be done privately.
-    - It is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
+    - When an answer given is in question, it is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
```

Comment:
I added this back in verbatim, since otherwise "its content" had nothing to modify. 

I think a v-next proposal will be removing this...I don't like AJs superseding QMs.

Reply:
Agreed. Also, I think we should expand (in v-next) about QMs having ultimate authority, fully subjugating AJs to QMs.